ipaclient/install/client.py

```diff
@@ -3632,6 +3641,20 @@ def uninstall(options):
         except Exception:
             pass
 
+    # Restore unbound to its original status
+    if statestore.restore_state("dns_over_tls", "enabled"):
+        unbound = services.knownservices['unbound']
+        if not statestore.restore_state('unbound', 'running'):
+            unbound.stop()
+        if not statestore.restore_state('unbound', 'enabled'):
+            unbound.disable()
+        # restore unbound config files that were removed during IPA install
+        remove_file(paths.UNBOUND_CONF)
```

Comment:
UNBOUND_CONF isn't backed up but is unconditionally removed on uninstall. Is that expected? Same question on the server side.

Reply:
OK this works for me then, no other objections.

Reply:
There is a failure in the temp commit related to fstore